src/interface/term.ml

```diff
@@ -681,7 +683,10 @@ module type Ae_Base = sig
   val distinct : t list -> t
   (** Distinct constraints on terms. *)
 
-  val in_interval : t -> bool * bool -> t -> t -> t
+  val multi_trigger : t list -> t
+  (** Create a multi trigger from a list of arbtirary terms. *)
+
+  val semantic_trigger : t -> t
   (** Semantic trigger: "in interval" check. *)
```

Comment:
This should either be renamed back to `"in interval"` or the documentation updated to indicate that this is a marker term for semantic triggers (which may or may not be "in interval" checks).